google/cloud/storage/bucket.py

```diff
@@ -1725,6 +1729,11 @@ def delete_blobs(
         :param client: (Optional) The client to use.  If not passed, falls back
                        to the ``client`` stored on the current bucket.
 
+        :type preserve_generation: bool
+        :param preserve_generation: (Optional) Preserves blob generation if set to True. A list of
```

Comment:
Suggested wording: "Deletes only the generation specified on the blob object, instead of the live version, if set to True. Only :class:`~google.cloud.storage.blob.Blob` objects can have their generation set in this way."